index.js

```diff
@@ -18,3 +18,12 @@ module.exports = function (ms) {
 
 	return thunk;
 };
+
+module.exports.reject = function (ms, value) {
```

Comment:
:+1: See https://github.com/ariporad/delay/pull/2, I think it covers all the scenarios. Feel free to steel whatever you like from that (no need for a merge / commit credit - it was too quick and dirty to be deserving of that).